modules/ssh-agent-using.adoc

```diff
@@ -150,7 +150,10 @@ $ eval "$(ssh-agent -s)"
 ----
 Agent pid 31874
 ----
-
+[NOTE]
```

Comment:
So two things on where this is moved:

1. This needs a "+" on the line above this, in order to keep the note indented with this step. If you look at the rendered build, you can see that the step below has its numbering restarted at 1.
2. If you're adding this to a step, I think it probably belongs under step 1, because the `ssh-gen` command is where the key is actually being generated, not this step (which is starting the ssh-agent).